repository/src/main/java/org/apache/atlas/repository/store/graph/v2/ClassificationAssociator.java

```diff
@@ -166,6 +167,9 @@ public void setClassifications(Map<String, AtlasEntityHeader> map, boolean overr
                 AtlasEntityHeader incomingEntityHeader = map.get(guid);
                 String typeName = incomingEntityHeader.getTypeName();
                 AtlasEntityHeader entityToBeChanged;
+                AtlasVertex assetVertex = AtlasGraphUtilsV2.findByGuid(this.graph, guid);
```

Comment:
Review all APIs defined in EntityRest.java for tags & BM updates & ensure you have handled all cases